morpheus/pipeline/receiver.py

```diff
@@ -99,17 +100,15 @@ def get_input_pair(self) -> StreamPair:
                 self._is_linked = True
             else:
                 # We have multiple senders. Create a dummy stream to connect all senders
+                self._input_stream = builder.make_node_component(
+                    self.parent.unique_name + f"-reciever[{self.port_number}]", mrc.core.operators.map(lambda x: x))
```

Comment:
Why not put the unique name in the string as well?
```suggestion
                    f"{self.parent.unique_name}-reciever[{self.port_number}]", mrc.core.operators.map(lambda x: x))
```